README.md

```diff
@@ -59,6 +59,21 @@ This architecture ensures that your content remains exclusively on your device t
 
 6. Click the puzzle piece icon in Chrome toolbar and pin LLMFeeder for easy access
 
+### Option 3: Load in Firefox (Manual)
```

Comment:
Let's update this to have 2 sections and we will probably need to remove "Chrome" badging from other places as well in the readme. Probably add "cross-browser" extension and then have a heading for Chrome and for Firefox. 

WDYT?

Reply:
Yes agreed, will do it 😄